src/Sulu/Component/Rest/ListBuilder/ListRestHelper.php

```diff
@@ -22,6 +22,13 @@
  */
 class ListRestHelper implements ListRestHelperInterface
 {
+    const PARAMETER_FIELDS = 'fields';
+    const PARAMETER_PAGE = 'page';
+    const PARAMETER_SEARCH = 'search';
+    const PARAMETER_SEARCH_FIELDS = 'searchFields';
+    const PARAMETER_SORT_BY = 'sortBy';
+    const PARAMETER_SORT_ORDER = 'sortOrder';
```

Comment:
I think I would not do it... because then somebody could implement another `ListRestHelper` using a different names.